tests/i18n/test_fr.py

```diff
@@ -0,0 +1,83 @@
+"""Test French validators."""
+
```

Comment:
```diff

+ # standard
+ from typing import Union

# external
```

Use [`Union`](https://docs.python.org/3/library/typing.html#typing.Union) because validators currently supports Python 3.8 & 3.9.

Ref: https://peps.python.org/pep-0604/